src/cleanfigure.m

```diff
@@ -206,8 +203,9 @@ function pruneOutsideBox(meta, handle)
   %hasMarkers = ~strcmp(marker,'none');
   hasLines = true;
```

Comment:
To be honest, I have no clue why those lines are commented out. This seems like somebody went debugging in there and forgot to switch them back. We should `git blame` those to see what happened.